6.0/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys E3036906AD9F30807351F
 
 # https://doc.owncloud.org/server/8.1/admin_manual/installation/source_installation.html#prerequisites
 RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
-	&& docker-php-ext-install gd intl mbstring mcrypt mysql pdo_mysql pdo_pgsql pgsql zip
+	&& docker-php-ext-install gd intl mbstring mcrypt mysql pdo_mysql pdo_pgsql pgsql zip \
+	&& a2enmod rewrite
```

Comment:
Can we get the `a2enmod rewrite` on a separate line like [`wordpress` does](https://github.com/docker-library/wordpress/blob/ea9a5ca1de449706b2015b3337c1ba0a70dd856a/apache/Dockerfile#L3)?